src/fn_call.rs

```diff
@@ -560,6 +560,17 @@ pub trait EvalContextExt<'a, 'mir, 'tcx: 'a + 'mir>: crate::MiriEvalContextExt<'
                 let n = this.memory().get(ptr.alloc_id)?.read_c_str(tcx, ptr)?.len();
                 this.write_scalar(Scalar::from_uint(n as u64, dest.layout.size), dest)?;
             }
+            "cbrt" => {
+                let f = this.read_scalar(args[0])?.to_f64()?;
```

Comment:
Converting to host floats will be a bit harder once https://github.com/rust-lang/rust/pull/61673 lands. See https://github.com/rust-lang/miri/pull/764.